.github/workflows/frontend.yml

```diff
@@ -162,7 +191,7 @@ jobs:
       - name: Setup conda environment with python 3.9 and R
         uses: conda-incubator/setup-miniconda@v3
         with:
-          activate-environment: python_3_with_R
+          activate-environment: python_only
```

Comment:
Could you share the reason for this change?

Reply:
It was an incorrect change, so I rolled it back. Thank you for taking a look.